GithubHandlers.py

```diff
@@ -49,6 +50,17 @@ def __init__(
         self.prs_nums = [pr.number for pr in self.prs]
         self.prs_dict = {k: v for k, v in zip(self.prs_nums, self.prs)}
 
+    def index_pr(self, pr):
```

Comment:
Verify if the `index_pr` method could impact performance for large PRs and consider optimization.